src/i18n/en.js

```diff
@@ -37,6 +37,7 @@ export default {
     control_panel_url: `${urlControlPanelMain}`,
     copyright_MD: `© ${year} Doppler LLC. All rights reserved. [Privacy Policy & Legals](${urlPrivacy}).`,
     empty_data: `Your domain has no data to show in this section yet.`,
+    general_error: `Ouch! An unexpected error occurred, try again`,
```

Comment:
What do you think about calling this _common.unexpected_error_?

```suggestion
    unexpected_error: `Ouch! An unexpected error occurred, try again`,
```

Reply:
How about: `Ouch! An unexpected error occurred, please try again,`?

Reply:
A request in english without please sounds rude.

Reply:
changed! thanks